i18n-helpers/src/bin/mdbook-xgettext.rs

```diff
@@ -109,17 +266,31 @@ fn main() -> anyhow::Result<()> {
         .config
         .get_renderer("xgettext")
         .ok_or_else(|| anyhow!("Could not read output.xgettext configuration"))?;
-    let path = cfg
-        .get("pot-file")
-        .ok_or_else(|| anyhow!("Missing output.xgettext.pot-file config value"))?
-        .as_str()
-        .ok_or_else(|| anyhow!("Expected a string for output.xgettext.pot-file"))?;
```

Comment:
I love getting rid of this, but please update the command line here

https://github.com/google/mdbook-i18n-helpers/blob/1480904ad45f2dab4949d30826335a0a7a3b96fb/i18n-helpers/USAGE.md?plain=1#L79-L82

as well.

It would be good to also mention something about the `depth` parameter there.